k9mail-library/src/main/java/com/fsck/k9/mail/internet/UnsupportedContentTransferEncodingException.java

```diff
@@ -0,0 +1,7 @@
+package com.fsck.k9.mail.internet;
+
+public class UnsupportedContentTransferEncodingException extends Exception {
```

Comment:
Why not make this a subclass of `MessagingException` so we don't need to change all `throws` clause?